### PR TITLE
LevelUX bugs

### PR DESCRIPTION
**Pull request by shubhi1092**
- Language-specific short description not working in command bank
- Change default type for description and short description to be generic for all languages for the ease of content creation
- Play level view loading error in case of editor mode (i.e. query param dev=true)(because of the changes in loading view for ozaria and the new intro screen)

### CODE REVIEW

File: app/schemas/models/level_component.coffee [hunk 1]
```diff
@@ -34,6 +34,7 @@ PropertyDocumentationSchema = c.object {
   type: c.shortString(title: 'Type', description: 'Intended type of the property.')
   shortDescription:
     oneOf: [
+      {title: 'Short Description', type: 'string', description: 'Short Description of the property.', maxLength: 1000, format: 'markdown'}
```
Comment (AndrewJakubowicz) [inline app/schemas/models/level_component.coffee]: Why was this moved?

File: ozaria/site/views/play/level/PlayLevelView.js [hunk 2]
```diff
@@ -884,17 +884,9 @@ class PlayLevelView extends RootView {
       this.surface.showLevel()
     }
     Backbone.Mediator.publish('level:set-time', { time: 0 })
-    if (
```
Comment (AndrewJakubowicz) [inline ozaria/site/views/play/level/PlayLevelView.js]: I'm a little confused regarding why this could be deleted. But I also love seeing code get deleted!

File: app/schemas/models/level_component.coffee (hunk 1 above)
Comment (shubhi1092) [reply app/schemas/models/level_component.coffee]: Yep this was moved so that while adding short description from the level editor, by default the type `string` is selected (i.e. generic description for all languages) instead of language-specific description object. This is done for the ease of content creation because most of the time there will be generic short description, and can manually change it to language-specific type if required. (Same for the property below)

File: ozaria/site/views/play/level/PlayLevelView.js (hunk 2 above)
Comment (shubhi1092) [reply ozaria/site/views/play/level/PlayLevelView.js]: Yep this solves the issue with dev=true. Basically, the code in the if-block was no longer relevant because this was to skip the legacy intro screen when in dev mode, but now we are keeping the behavior consistent. Also `loadingView.startUnveiling` has been removed since that code is moved to the on click handler of `start` button on the intro modal, i.e. publish of this event: https://github.com/codecombat/codecombat/pull/5505/files#diff-5a00df3717cc9ade14619a5f033f1c9bR900.
